test/edu-benefits/pages/contactInformation.unit.spec.jsx

```diff
@@ -53,7 +53,7 @@ describe('Edu pages contactInformation', () => {
     const formDOM = findDOMNode(form);
     submitForm(form);
 
-    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(6);
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(5);
```

Comment:
I haven't investigated it thoroughly yet, but this is because it was being validated with previous information. I just fixed up a bug where the validation was run with the data before it was changed. That made this go back to 6.

I'm guessing the `data={{}}` sets the country to blank, overriding the default..? That'd cause there to be 6 errors instead of 5.